docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,147 @@
+needs to have the ``atop`` tool installed locally.
+
+The system utilization statistics are internally read from
+``/var/log/atop/atop_YYYYMMDD`` files which correspond to the each day in the last
+``atop.max-history-days`` days. These files contain compressed binary data written
+on a ``10`` minutes interval by ``/etc/atop/atop.daily`` cron daemon.
+
+Connector configuration
+-----------------------
+
+Create ``etc/catalog/system_monitor.properties``
+
+.. code-block:: text
+
+    connector.name=atop
+    atop.executable-path=/usr/bin/atop
+
+Configuration
+-----------------------
+
+.. list-table:: Configuration variables
+  :widths: 40, 10, 10, 40
+  :header-rows: 1
+
+  * - Property name
+    - Default value
+    - Required
+    - Description
+  * - ``atop.concurrent-readers-per-node``
+    - ``1``
+    - Yes
+    - The number of concurrent read operations allowed per node.
+  * - ``atop.executable-path``
+    - (none)
+    - Yes
+    - The file path on the local file system for the ``atop`` utility
+  * - ``atop.executable-read-timeout``
+    - ``1ms``
+    - Yes
+    - The timeout when reading from the atop process.
+  * - ``atop.max-history-days``
+    - ``30``
+    - Yes
+    - The maximum number of days behind to take into account statistics.
+  * - ``atop.security``
+    - ``ALLOW_ALL``
+    - Yes
+    - The authorization rules for the connector. If set to ``FILE``,
+      see :ref:`catalog-file-based-access-control`
+      for information on the authorization configuration file.
+  * - ``atop.time-zone``
+    - System default
+    - Yes
+    - The timezone ID in which the atop data is collected. Generally the timezone of the host.
+
+Usage
+-----
+
+The Atop connector provides a ``default`` schema.
+
+The tables available exposed by this connector can be retrieved by running ``SHOW TABLES``::
+
+    SHOW TABLES FROM system_monitor.default;
+
+.. code-block:: text
+
+      Table
+    ---------
+     disks
+     reboots
+    (2 rows)
+
+
+The ``disks`` table offers disk utilization statistics recorded on the Trino node.
+
+.. list-table:: Disks columns
+  :widths: 30, 30, 40
+  :header-rows: 1
+
+  * - Name
+    - Type
+    - Description
+  * - ``host_ip``
+    - ``varchar``
+    - Trino worker machine IP
+  * - ``start_time``
+    - ``timestamp(3) with time zone``
+    - Interval start time for the statistics
+  * - ``end_time``
+    - ``timestamp(3) with time zone``
+    - Interval end time for the statistics
+  * - ``device_name``
+    - ``varchar``
+    - Logical volume/hard disk name
+  * - ``utilization_percent``
+    - ``double``
+    - The percentage the unit was busy handling requests
+  * - ``io_time``
+    - ``interval day to second``
+    - Time spent for I/O
+  * - ``read_requests``
+    - ``bigint``
+    - Number of reads issued
+  * - ``sectors_read``
+    - ``bigint``
+    - Number of sectors transferred for reads
+  * - ``write_requests``
+    - ``bigint``
+    - Number of writes issued
+  * - ``write_requests``
```

Comment:
write_requests occurs twice. I assume you meant sectors_write or something?

Reply:
`sectors_written`